### PR TITLE
percona-toolkit 2.2.14

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/percona-toolkit.rb [hunk 1]
```diff
@@ -15,15 +16,15 @@ class PerconaToolkit < Formula
   depends_on "openssl"
 
   resource "DBD::mysql" do
-    url "http://search.cpan.org/CPAN/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.027.tar.gz"
-    mirror "http://search.mcpan.org/CPAN/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.027.tar.gz"
-    sha1 "3bf1edd6f0b4f6144b2aaa715c80df3fb1cd2119"
+    url "http://www.cpan.org/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.032_01.tar.gz"
+    mirror "http://search.mcpan.org/CPAN/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.032_01.tar.gz"
```
Comment (vszakats) [inline Library/Formula/percona-toolkit.rb]: I suggest this new SSL/TLS URL for `mirror` (above one redirects to it):
https://cpan.metacpan.org/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.032_01.tar.gz

File: Library/Formula/percona-toolkit.rb [hunk 2]
```diff
@@ -15,15 +16,15 @@ class PerconaToolkit < Formula
...
+    sha256 "76756b24eed46553f9dad22d0682a82b50ca2c8500ea4ede0a414acab48c9e77"
   end
 
   resource "JSON" do
     url "http://search.cpan.org/CPAN/authors/id/M/MA/MAKAMAKA/JSON-2.90.tar.gz"
     mirror "http://search.mcpan.org/CPAN/authors/id/M/MA/MAKAMAKA/JSON-2.90.tar.gz"
```
Comment (vszakats) [inline Library/Formula/percona-toolkit.rb]: And this one instead of the above:
https://cpan.metacpan.org/authors/id/M/MA/MAKAMAKA/JSON-2.90.tar.gz

**Comment by vszakats** [comment]
Thank you!

**Comment by xu-cheng** [comment]
Thanks

**Pull request closed without merging**